src/roc-zenodo/getZenodoDeposition.js

```diff
@@ -67,10 +69,20 @@ function validateCreators(creators) {
     if (typeof creator.affiliation !== 'string') {
       throw new TypeError('creator must have an affiliation');
     }
-    toReturn.push({
+
+    const creatorToReturn = {
       name: creatorName,
       affiliation: creator.affiliation,
-    });
+    };
+
+    if ('orcid' in creator) {
+      if (okOrcid(creator.orcid)) {
+        creatorToReturn.orcid = creator.orcid;
+      } else {
+        throw new TypeError('Not a valid ORCID identifier');
```

Comment:
Should we include the value of the orcid identifier in the error message? That might be useful in case there are multiple identifiers in the same publication to know which one is wrong.